page-changelog.php

```diff
@@ -7,14 +7,59 @@
 <li><a href="#latest10.0">ownCloud 10.0</a></li>
 <li><a href="#latest9.1">ownCloud 9.1</a></li>
 <li><a href="#latest9.0">ownCloud 9.0</a></li>
-<li><a href="#latest8.2">ownCloud 8.2</a></li>
+<li><a href="#latest8.2">ownCloud 8.2 (unsupported!)</a></li>
 <li><a href="#latest8.1">ownCloud 8.1 (unsupported!)</a></li>
 <li><a href="#latest8.0">ownCloud 8.0 (unsupported!)</a></li>
 <li><a href="#latest7">ownCloud 7 (unsupported!)</a></li>
 <li><a href="#latest6">ownCloud 6 (unsupported!)</a></li>
 </ul>
 
 <a name="latest10.0"></a></li>
+<h3>Version 10.0.2 <small>May 30 2017</small></h3></li>
+Release notes: <a href="https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-2">Release notes</a></br>
+<ul>
+<li>[major] Fix issue with database.xml migration being triggered twice on market app install: <a href="https://github.com/owncloud/core/issues/27982">core/#27982</a></li>
+<li>[major] Apps formerly marked as shipped can now be uninstalled - <a href="https://github.com/owncloud/core/issues/27985">core/#27985</a></li>
+<li>[major] Market now properly updates app version when using multiple apps paths: <a href="https://github.com/owncloud/core/issues/27989">core/#27989</a></li>
```

Comment:
Ending ``:`` where other entries are using `` - `` as a limiter between the text and the github issue